aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -576,9 +601,26 @@ module aptos_framework::staking_contract {
         );
     }
 
+    /// Allows an operator to change its beneficiary.
+    public entry fun set_beneficiary_for_operator(operator: &signer, new_beneficiary: address) acquires Beneficiary {
+        let operator_addr = signer::address_of(operator);
```

Comment:
Can you leave a comment to document that we don't need to validate the operator here as this is a resource stored on the operator account and not at the staking contract level?

Reply:
Done.